### ISSUE TITLE
Invalid cepces.conf content

### ISSUE THREAD

**Issue by adam-kosseck** [openSUSE/cepces#23]
Attempting to generate a certificate request with the following command results in an error.
```
# getcert request -c cescep-ca -w -v -M 644 -T Machine -I $(hostname) -k /root/computer.key -f /root/computer.crt
New signing request "hostname_removed" added.
State NEWLY_ADDED_READING_KEYINFO, stuck: no.
State GENERATING_KEY_PAIR, stuck: no.
State NEED_KEY_GEN_PERMS, stuck: yes.
```

Error found in /var/log/messages
```
<TIME> <HOST>certmonger[<PID>]: <TIME> cepces.config.Configuration:DEBUG:Initializing application configuration.
<TIME> <HOST>certmonger[<PID>]: <TIME> cepces.config.Configuration:DEBUG:Reading: /etc/cepces/cepces.conf
<TIME> <HOST>certmonger[<PID>]: <TIME> __main__:ERROR:Traceback (most recent call last):
<TIME> <HOST>certmonger[<PID>]:  File "/usr/local/libexec/certmonger/cepces-submit", line 64, in main
<TIME> <HOST>certmonger[<PID>]:    krb5_overrides=krb5_overrides)
<TIME> <HOST>certmonger[<PID>]:  File "/usr/lib/python3.6/site-packages/cepces/config.py", line 141, in load
<TIME> <HOST>certmonger[<PID>]:    config['global'][key] = val
<TIME> <HOST>certmonger[<PID>]:  File "/usr/lib64/python3.6/configparser.py", line 1237, in __setitem__
<TIME> <HOST>certmonger[<PID>]:    self._parser._validate_value_types(option=key, value=value)
<TIME> <HOST>certmonger[<PID>]:  File "/usr/lib64/python3.6/configparser.py", line 1177, in _validate_value_types
<TIME> <HOST>certmonger[<PID>]:    raise TypeError("option values must be strings")
<TIME> <HOST>certmonger[<PID>]: TypeError: option values must be strings
```

**Environment:**
- RHEL 8.6
-- SELinux was disabled as a troubleshooting step
- Certmonger 0.79.13-5 (from EPEL 8 RPM)
- Cepces 0.3.5-6 (from EPEL 8 RPM)

Certmonger configuration file has not been modified from default.
Cepces CA was created with:
`getcert add-ca -c cescep-ca -e '/usr/local/libexec/certmonger/cepces-submit --server=${PKI_SERVER} --keytab=/etc/krb5.keytab'`

Entry looks valid:
```
# getcert list-cas
...
CA 'cescep-ca':
        is-default: no
        ca-type: EXTERNAL
        helper-location: /usr/local/libexec/certmonger/cepces-submit --server=<pki server fully qualified hostname, lowercase> --keytab=/etc/krb5.keytab
```

/etc/cepces/cepces.conf file was created like this:
```
mkdir -p /etc/cepces
cp /usr/local/etc/cepces/cepces.conf.dist /etc/cepces/cepces.conf
sed -i "s/server=.*/server=${PKI_SERVER}/" /etc/cepces/cepces.conf
sed -i "s/^#realm=.*/realm=${DOMAIN^^}/" /etc/cepces/cepces.conf
``` 

/etc/cepces/cepces.conf contents (minus comments:
```
[global]
server=<pki server fully qualified hostname, lowercase>
type=Policy
auth=Kerberos
endpoint=https://${server}/ADPolicyProvider_CEP_${auth}/service.svc/CEP
poll_interval=3600

[kerberos]
realm=<fully qualified domain name, uppercase>
ccache=True

principals=
  ${shortname}$$
  ${SHORTNAME}$$
  host/${SHORTNAME}
  host/${fqdn}

enctypes=
  des-cbc-crc
  des-cbc-md5
  arcfour-hmac
  aes128-cts-hmac-sha1-96
  aes256-cts-hmac-sha1-96

[certificate]
```

Am I doing something wrong, or is this a bug?

**Comment by dmulder** [comment]
I've reproduced this. I'll track down the issue.

**Comment by dmulder** [comment]
It's because the new parameter to cepces-submit 'poll_interval' has no default value.

**Comment by dmulder** [comment]
This is fixed in #24.
